packages/status-js/src/client/client.ts

```diff
@@ -51,13 +56,21 @@ class Client {
 
   static async start(options: ClientOptions) {
     // Waku
+    const fleet =
+      options.environment === 'production'
```

Comment:
The default environment should be production.

Reply:
🤔

Reply:
Done at https://github.com/status-im/status-web/pull/294/commits/2bdc224320be9d5cf44f6cde0f62a70691101ca7